### ISSUE TITLE
Why are there no log file for one machine?

### ISSUE THREAD

**Issue by TccccD** [diux-dev/imagenet18#7]
Could you provide log file for one machine?

**Comment by bearpelican** [comment]
I've added the raw logs here as a benchmark
https://github.com/diux-dev/imagenet18/commit/f0fc7133b3527caf38ba1234b5326646fe100034

**Comment by yaroslavvb** [comment]
BTW, logs with all event metrics we collect are are linked in train.py above each run, including 1-machine
IE

```
# 109:12 to 93.00
# events: https://s3.amazonaws.com/yaroslavvb/logs/imagenet-1
# logs: https://s3.amazonaws.com/yaroslavvb/logs/imagenet1.tar
```
You can follow instructions like in https://github.com/diux-dev/imagenet18/issues/5 to convert them into TSV file

**Comment by TccccD** [comment]
This link does not reach @yaroslavvb

**Comment by TccccD** [comment]
Is there a log of the following ?
```
lr = 1.0
scale_224 = 224/512
scale_288 = 128/512
one_machine = [
  {'ep':0,  'sz':128, 'bs':512, 'trndir':'-sz/160'},
  {'ep':(0,5),  'lr':(lr,lr*2)}, # lr warmup is better with --init-bn0
  {'ep':5, 'lr':lr},
  {'ep':14, 'sz':224, 'bs':224,
                'lr':lr*scale_224},
  {'ep':16,     'lr':lr/10*scale_224},
  {'ep':27,     'lr':lr/100*scale_224},
  {'ep':32, 'sz':288, 'bs':128, 'min_scale':0.5, 'rect_val':True,
                'lr':lr/100*scale_288},
  {'ep':(33,35),'lr':lr/1000*scale_288}
]
```

**Comment by TccccD** [comment]
This link still can not reach @yaroslavvb
got:
```
<Error>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Key>logs/imagenet1.tar</Key>
<RequestId>644A92E3CFA015F0</RequestId>
<HostId>
o1rrk6jiBwhjFgZ6Eb+CxPlmS5f8X/g+Qy7HET/dKZVTiW+u15GqXSacIR/u9w0s6Q+P5QJDHXk=
</HostId>
</Error>
```

**Comment by yaroslavvb** [comment]
tldr: link to events is this: https://s3.amazonaws.com/yaroslavvb/logs/imagenet-1
